api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/components/steps/ErrorStep.java

```diff
@@ -0,0 +1,68 @@
+package edu.cornell.mannlib.vitro.webapp.dynapi.components.steps;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+
+import edu.cornell.mannlib.vitro.webapp.dynapi.components.OperationResult;
+import edu.cornell.mannlib.vitro.webapp.dynapi.components.Parameters;
+import edu.cornell.mannlib.vitro.webapp.dynapi.components.ProcedureDescriptor;
+import edu.cornell.mannlib.vitro.webapp.dynapi.computation.StepInfo;
+import edu.cornell.mannlib.vitro.webapp.dynapi.data.DataStore;
+import edu.cornell.mannlib.vitro.webapp.dynapi.data.conversion.InitializationException;
+import edu.cornell.mannlib.vitro.webapp.utils.configuration.Property;
+
+public class ErrorStep implements Step{
```

Comment:
what is the case study for using ErrorStep? Can we attach some operation within ErrorStep for handling errors? Anyway, please rename setMaxLenght method to setErrorCode(int errorCode)

Reply:
Logical error, specified in procedure description. [Example](https://github.com/litvinovg/Vitro/blob/13141ae48cb220f3ce2e95f4137a0edbfbe2a8ca/home/src/main/resources/rdf/dynapiAbox/everytime/endpoint_procedure_execute_report_generator.n3#L65).
The method has been renamed.
I think we can extend it in future, on the other hand it would be just an operational step then.